pkg/api/v1/defaults_test.go

```diff
@@ -1332,3 +1332,110 @@ func TestSetDefaultHostPathVolumeSource(t *testing.T) {
 		t.Errorf("Expected v1.HostPathVolumeSource default type %v, got %v", expectedType, defaultType)
 	}
 }
+
+func TestSetDefaults_SecurityContext(t *testing.T) {
+	var nonRoot int64 = 1000
+	var root int64 = 0
+	pfalse := false
+	ptrue := true
+
+	tests := map[string]struct {
+		sc     v1.SecurityContext
+		expect bool
+	}{
+		"allowPrivilegeEscalation nil security context nil": {
+			expect: true,
+		},
+		"allowPrivilegeEscalation nil capAddSysadmin": {
```

Comment:
s/nil capAddSysadmin/and CAP_SYS_ADMIN/

Reply:
Ah, I see the logic in the names now... Could you make it more explicit? Otherwise it could be misinterpreted which exactly value is `nil` in the cases like `allowPrivilegeEscalation nil privileged`